test/e2e/antreapolicy_test.go

```diff
@@ -3193,6 +3195,12 @@ func testFQDNPolicy(t *testing.T) {
 		},
 		{
 			Pod(namespaces["y"] + "/b"),
+			"stackoverflow.com",
+			80,
+			Dropped,
+		},
+		{
+			Pod(namespaces["z"] + "/a"),
```

Comment:
nit: can we make it clear in comments that these Pods are randomly chosen?

Reply:
Done.